tests/data/outputs/events.txt

```diff
@@ -0,0 +1 @@
+sendfile,recvfrom,readv,sendto,send,read,recvmmsg,write,recvmsg,pwrite,sendmmsg,sendmsg,pread,writev,recv,pwritev,preadv
```

Comment:
@jasondellaluce my list is matching with the latest list.

Reply:
>What error are you encountering when compiling Falco? Either way, you can install locally the latest dev version by downloading it from https://download.falco.org/?prefix=packages/bin-dev/. Then, once installed, the test suite should run successfully by following the readme instructions.

There was some deps issue but now I've solved it. Used the latest(master) `falco` for the test  and it has since been passing.